pkg/sfu/videolayerselector/base.go

```diff
@@ -0,0 +1,120 @@
+package videolayerselector
```

Comment:
`Base` video layer selector which is used by all implementations. Basically holds all state and also implements most methods which is mostly common.